src/components/VtkTwoView.vue

```diff
@@ -400,6 +407,48 @@ export default defineComponent({
       { immediate: true, deep: true }
     );
 
+    // --- arrows change slice --- //
+
+    const { currentImageID } = useCurrentImage();
+
+    const onKeyDown = (event: KeyboardEvent) => {
+      if (!currentImageID.value) return;
+
+      let sliceOffset = 0;
+      switch (event.key) {
+        case 'ArrowLeft':
+          sliceOffset = -1;
+          break;
+        case 'ArrowRight':
+          sliceOffset = 1;
+          break;
+        case 'ArrowUp':
+          sliceOffset = -1;
+          break;
+        case 'ArrowDown':
+          sliceOffset = 1;
+          break;
+        default:
+          break;
+      }
+      if (sliceOffset) {
+        viewConfigStore.updateSliceConfig(viewID.value, currentImageID.value, {
+          slice: currentSlice.value + sliceOffset,
+        });
+
+        event.stopPropagation();
+      }
+    };
+
+    const hover = ref(false);
+    const onDocumentKeyDown = (event: KeyboardEvent) => {
+      if (hover.value) onKeyDown(event);
+    };
+    document.addEventListener('keydown', onDocumentKeyDown);
+    onBeforeUnmount(() => {
+      document.removeEventListener('keydown', onDocumentKeyDown);
+    });
```

Comment:
I would recommend using [onKeyStroke](https://vueuse.org/core/onKeyStroke/). We have vue-use as a dependency, which can help cut down on having to define cleanup ourselves.

Reply:
Excellent, done.